target/linux/ath79/image/generic.mk

```diff
@@ -542,7 +590,9 @@ TARGET_DEVICES += ocedo_ursus
 
 define Device/openmesh_om5p-ac-v2
   ATH_SOC := qca9558
-  DEVICE_TITLE := OpenMesh OM5P-AC v2
+  DEVICE_VENDOR := Open-Mesh
```

Comment:
Sure about that change? Additionally, this would not match node name then "openmesh_..."